sample-hip/src/main/java/com/nha/abdm/hip/PatientController.java

```diff
@@ -1,61 +1,37 @@
-package com.nha.abdm.wrapper;
+package com.nha.abdm.hip;
 
 import com.nha.abdm.wrapper.client.api.LinkApi;
 import com.nha.abdm.wrapper.client.api.PatientsApi;
 import com.nha.abdm.wrapper.client.invoker.ApiException;
-import com.nha.abdm.wrapper.client.model.CareContext;
-import com.nha.abdm.wrapper.client.model.FacadeResponse;
-import com.nha.abdm.wrapper.client.model.LinkCareContextsRequest;
-import com.nha.abdm.wrapper.client.model.Patient;
-import com.nha.abdm.wrapper.client.model.PatientWithCareContext;
-import com.nha.abdm.wrapper.client.model.VerifyOTPRequest;
+import com.nha.abdm.wrapper.client.model.*;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
 
-public class Main {
+@RestController
+@RequestMapping(path = "/v1")
+public class PatientController {
 
-    private static final String requestId = "260ad625-ffb9-4c7d-b5bc-e099577e7e87";
-    public static void main(String[] args) throws ApiException, InterruptedException {
-        /**
-         * Uncomment below to test addition or modification of patients.
-         */
-        // addOrModifyPatients();
+    private static final String requestId = "260ad625-ffb9-4c7d-b5bc-e099577e7e88";
 
-        LinkApi linkApi = new LinkApi();
-        /**
-         * Uncomment below to test linking of care-contexts.
-         */
-        // linkCareContextRequest(linkApi, requestId);
-
-        /**
-         * Uncomment below to test linking of care-contexts.
-         */
-        // This is just an illustraion. Verify OTP should be called once HIP gets OTP from patient.
-        // verifyOtp(linkApi, requestId, 123456);
-
-        /**
-         * Uncomment below to test status of link request.
-         */
-        /**
-        long NANOSEC_PER_SEC = 1000l*1000*1000;
+    @GetMapping({"/patients/{patientId}"})
+    public Patient fetchCareContextStatus(@PathVariable String abhaAddress) {
```

Comment:
nit: Can the function name be `fetchPatientById`?

Reply:
Since all the other API paths are starting with `test-wrapper`, can we modify the path of this API as well, may be prefix it at the Controller level itself `v1/test-wrapper`.

Reply:
But this is not wrapper's api. This is the api which HIP needs to implement and expose.

Reply:
changed the method name, thanks